docs/tutorials/start-a-private-network-with-substrate.md

```diff
@@ -145,77 +118,107 @@ yarn
-  --port 30333 \
+  --bob \
+  --port 30334 \
   --telemetry-url ws://telemetry.polkadot.io:1024 \
   --validator \
-  --name BobsNode \
   --bootnodes /ip4/<Alices IP Address>/tcp/<Alices Port>/p2p/<Alices Node ID>
 ```
 Most of these options are already explained above, but there are a few tripping points.
 * If these two nodes are running on the same physical machine, Bob MUST specify a different `--base-path` and `--port`.
 * Bob has added the `--bootnodes` flag and specified a single boot node, namely Alice's. He must correctly specify these three pieces of information which Alice can supply for him.
   * Alice's IP Address in the form `192.168.1.1`
   * Alice's Port, probably 30333
-  * Alice's node ID, copied from her log output. (`QmNTx7qYd5JiasUTzjaAbTCUeH5ot9xMjRePRjkSEdD7MY` in the output above.)
+  * Alice's node ID, copied from her log output. (`QmViTYm8Yr4yneyRGov9L87vLeVLoySFduVRNQ1d33WdXN` in the example output above.)
 
 If all is going well, after a few seconds, the nodes should peer together and start producing blocks. You should see some lines like
 
 ```
-Idle (1 peers), best: #1 (0x9f1b…9b57), finalized #0 (0x28be…45e5), ⬇ 1.7kiB/s ⬆ 1.4kiB/s
+Idle (1 peers), best: #1 (0x9f1b…9b57), finalized #1 (0x28be…45e5), ⬇ 1.7kiB/s ⬆ 1.4kiB/s
 ```
 
-This line shows that Bob has peered with Alice (`1 peers`), they have produced a block (`best: #1 (0x9f1b…9b57)`), and the block is not finalized (`finalized #0 (0x28be…45e5)`). In this network, only the genesis block will ever be finalized because there is no finality gadget in the node-template. Finality is beyond the scope of this tutorial.
+This line shows that Bob has peered with Alice (`1 peers`), they have produced a block (`best: #1 (0x9f1b…9b57)`), and the block is  finalized (`finalized #1 (0x28be…45e5)`).
 
 Generate Your Own Keys
 ----------------------
-Now that we know the fundamentals and the command line options, it's time to generate our own keys rather than using the well-known Alice and Bob keys.
-
-Each person who wants to participate in the blockchain can generate their own key using the [subkey tool](ecosystem/subkey) or the Polkadot JS Apps UI that we used previously. In this tutorial, we'll use the UI.
+Now that we know the fundamentals and the command line options, it's time to generate our own keys rather than using the well-known Alice and Bob keys. Each person who wants to participate in the blockchain can generate their own key using the [subkey tool](ecosystem/subkey). Be sure to record all of the output from this section as you will need it later.
 
-Navigate to the "Accounts" tab, and choose "Add Account". For this kind of session key, we'll need to select Edwards Curve near the bottom, and copy-paste the public address and the raw seed.
-
-![Screenshot of key generation](https://cdn.discordapp.com/attachments/559788157239951376/583363719753367567/unknown.png)
+First generate a mnemonic and see the sr25519 key and address associated with it.
```

Comment:
How do you feel about "highlighting" they key types? Also, I think giving some context while generating the keys will help the reader.

```suggestion
First generate a mnemonic and see the `sr25519` key and address associated with it. This key will be used by BABE for block production.
```